modules/standard/Random.chpl

```diff
@@ -1053,6 +1054,7 @@ module Random {
     }
 
 
+    module PCGRandomLib {
```

Comment:
Ooh, is this our first sub-sub-module?  Sweeeet!  Can't wait to see it under Lydia's new docs rendering...!